downgrade

```diff
@@ -25,17 +25,20 @@ currently_installed() {
   '
 }
 
+# <package name> [package path] …
 present_packages() {
   local i=1
+  local pkgname="$1"
+  shift
 
   (($#)) || return 1
 
   gettext 'Available packages:'
   printf '\n\n'
 
   for entry; do
-    output_package "$((i++))" "$entry"
-  done
+    output_package "$((i++))" "$entry" "$pkgname"
+  done | column -s '	' -t -R 2,4,6
```

Comment:
What does this do?

Reply:
This creates space indented table (output_package just divides cells with tabs).
I added a comment to it.